_cards/onboarding.md

```diff
@@ -5,27 +5,28 @@ type: info
 order: 23
 permalink: /onboarding.html
 urls:
-  onboarding: "https://getok-master-k8vopl-prod.pathfinder.gov.bc.ca/api/auth/login"
+  onboarding: "https://getok.pathfinder.gov.bc.ca/getok"
 ---
 
 The following onboarding process has been tested and we are looking for feedback to make it even better
 
 1. A developer on your team must create an account by using their IDIR credentials to Login to [GetOK](#GETOK)
-  a. Click the _Register New App_ button to send the GETOK an email with your application acronym (short name) and your IDIR account name.
+  a. In the navigation bar, go to _Request Account_ or _My Applications_ and click the _Add a new Application_ button, fill out the form with your application acronym (short name) any additional comments.
```

Comment:
Missing an "and" before "any additional comments"?
Might be good for a quick description of what an acronym is... though hopefully that's not opening a can of worms. But the last person to request put "CHES" in there

Reply:
making the text fix, not going to worry about explaining the acronym.  if they don't read know what it is, or read the GETOK hover/help, I can't really help much here.  if there is a link to IRS application acronyms, that'd be good.